05/app.js

```diff
@@ -8,7 +8,32 @@ const stats = {
 };
 
 /* tutaj umieść swój kod */
+const pList = document.querySelectorAll('[data-id]');
 
+function countClick(e) {
+    e.preventDefault();
+    const hrefAttr = e.target.getAttribute('href');
+    const dataAttr = e.target.getAttribute('data-id');
```

Comment:
to pobranie wykonałbym wew. odpowiedniego if-a, aby nie wykonywać niepotrzebnych operacji :)